src/components/Trade/hooks/useTradeQuoteService.tsx

```diff
@@ -103,43 +94,32 @@ export const useTradeQuoteService = () => {
     action,
     amount,
     buyAsset,
-    buyTradeAsset,
     receiveAddress,
     sellAccountMetadata,
     selectedCurrencyToUsdRate,
     sellAsset,
-    sellTradeAsset,
     wallet,
     isSendMax,
+    sellAmountCryptoPrecision,
   ])
 
   // Update trade quote
-  useEffect(
-    () => swapperDispatch({ type: SwapperActionType.SET_VALUES, payload: { quote: tradeQuote } }),
-    [tradeQuote, swapperDispatch],
-  )
+  useEffect(() => updateQuote(tradeQuote), [tradeQuote, updateQuote])
 
   // Set slippage if the quote contains a recommended value, else use the default
   useEffect(
     () =>
-      swapperDispatch({
-        type: SwapperActionType.SET_VALUES,
-        payload: {
-          slippage: tradeQuote?.recommendedSlippage
-            ? tradeQuote.recommendedSlippage
-            : DEFAULT_SLIPPAGE,
-        },
-      }),
-    [tradeQuote, swapperDispatch],
+      updateSlippage(
+        tradeQuote?.recommendedSlippage ? tradeQuote.recommendedSlippage : DEFAULT_SLIPPAGE,
```

Comment:
Not introduced by this PR but could use logical OR instead of ternary

Reply:
I like it: https://github.com/shapeshift/web/pull/4010/commits/a9eb39f00f8e54f7d1a6487c98b479b881993930